.github/workflows/cd-syft-dev.yml

```diff
@@ -70,15 +70,15 @@ jobs:
         id: buildx
         uses: docker/setup-buildx-action@v3
 
-      - name: Install Azure CLI
-        run: |
-          curl -sL https://aka.ms/InstallAzureCLIDeb | sudo bash
-          az version
+      # - name: Install Azure CLI
+      #   run: |
+      #     curl -sL https://aka.ms/InstallAzureCLIDeb | sudo bash
+      #     az version
 
-      - name: Login to Azure CLI
-        uses: azure/login@v1
-        with:
-          creds: ${{ secrets.AZURE_CREDENTIALS_GITHUB_CI }}
+      # - name: Login to Azure CLI
+      #   uses: azure/login@v1
+      #   with:
+      #     creds: ${{ secrets.AZURE_CREDENTIALS_GITHUB_CI }}
```

Comment:
Q: Do we want to remove the commented section after a test run?

Reply:
will remove once it's clear everything is working as expected